java/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Place `.gitignore` at the project root and includes `.idea/` folder as well. Its an IDE specific folder.